aiozk/recipes/base_watcher.py

```diff
@@ -10,25 +10,25 @@
 log = logging.getLogger(__name__)
 
 
-def maybe_future(fut, loop):
+def maybe_future(fut):
     if inspect.isawaitable(fut):
-        asyncio.ensure_future(fut, loop=loop)
```

Comment:
`ensure_future` is not in deprecation list, only function that is coroutines are affected

Reply:
Yes, and `open_connection` too. Turned them back.